ql/src/java/org/apache/hadoop/hive/ql/optimizer/GenMapRedUtils.java

```diff
@@ -775,7 +775,7 @@ public static void setTaskPlan(Path path, String alias,
     if (topOp instanceof TableScanOperator) {
       try {
         Utilities.addSchemaEvolutionToTableScanOperator(
-          (StructObjectInspector) tt_desc.getDeserializer().getObjectInspector(),
+          (StructObjectInspector) tt_desc.getSerDe().getObjectInspector(),
```

Comment:
Nit: please rename tt_desc so that it follows the java naming conventions.

Reply:
Done.